packages/api/src/common/transformers/normalizeAddress.transformer.ts

```diff
@@ -10,6 +10,9 @@ export const normalizeAddressTransformer: ValueTransformer = {
     if (!hex) {
       return null;
     }
+    if (hexTransformer.from(hex) === "0x") {
```

Comment:
Why? Empty string doesn't look like a valid address

Reply:
0x00 and 0x are different values. Tx with empty `to` field is EVM deploy tx

Reply:
Then we should process 0x differently, earlier

Reply:
If you keep `getAddress(hexTransformer.from(hex))` wouldn't it just return a zero address?

Reply:
No, it returns an error. I have left it like this in order to be aligned with etherscan since they are returning empty string in cases like this. We can switch to 0x000....

Reply:
I don't have much context on the PR, but from what I've understood `0x` is like of special case for contract deployment (like, and "empty to field"), which is different from 0x00..00 (IIRC in early times it was used for contract deployment, but nowadays you just don't specify anything).